ethereumj-core/src/main/java/org/ethereum/net/rlpx/discover/MessageHandler.java

```diff
@@ -49,7 +49,11 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
     @Override
     public void channelRead0(ChannelHandlerContext ctx, DiscoveryEvent event) throws Exception {
-        nodeManager.handleInbound(event);
+        try {
+            nodeManager.handleInbound(event);
+        } catch (Throwable t) {
+            logger.error("Failed to process incoming message: {}", event.getMessage(), t);
```

Comment:
I would change log level to `WARN` or even `INFO`, since we can receive any garbage and this is not our error. 
I'd also output `t.toString()` instead of `t` to not flood logs with traces in case of invalid messages 

Ideally we can also try to differentiate the message handle problems onto `MalformedMessageException` and others. The first one is not error, the second one should be assumed as our error

Reply:
Decided to reduce logging level for the beginning. `MalformedMessageException` requires much more changes in the codebase.